evap/evaluation/tests/test_commands.py

```diff
@@ -0,0 +1,67 @@
+import datetime
+from django.core import management
+from unittest.mock import patch
+from django.test.testcases import TestCase
+from django.test.utils import override_settings
+from model_mommy import mommy
+
+from evap.evaluation.models import UserProfile, Course
+
+
+class TestUpdateCourseStateCommand(TestCase):
+    def test_update_courses_called(self):
+        with patch('evap.evaluation.models.Course.update_courses') as mock:
+            management.call_command('update_course_state')
+        mock.assert_called_once_with()
+
+
+@override_settings(REMIND_X_DAYS_AHEAD_OF_END_DATE=[0, 2])
+class TestSendRemindersCommand(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.today = datetime.date.today()
+
+        cls.user_to_remind = mommy.make(UserProfile)
+        cls.user_to_remind_twice = mommy.make(UserProfile)
+        cls.user_not_to_remind = mommy.make(UserProfile)
+
+        # Course evaluation already over.
+        cls.course_out_of_evaluation = mommy.make(
+                Course,
+                state='inEvaluation',
+                vote_start_date=cls.today - datetime.timedelta(days=2),
+                vote_end_date=cls.today - datetime.timedelta(days=1),
+                participants=[cls.user_not_to_remind])
+
+        # Course evaluation ends in 2 days.
+        cls.course1 = mommy.make(
+                Course,
+                state='inEvaluation',
+                vote_start_date=cls.today - datetime.timedelta(days=1),
+                vote_end_date=cls.today + datetime.timedelta(days=2),
+                participants=[cls.user_to_remind, cls.user_to_remind_twice, cls.user_not_to_remind],
+                voters=[cls.user_not_to_remind])
+
+        # Course evaluation ends today.
+        cls.course2 = mommy.make(
+                Course,
+                state='inEvaluation',
+                vote_start_date=cls.today - datetime.timedelta(days=2),
+                vote_end_date=cls.today,
+                participants=[cls.user_to_remind_twice],name='asdf')
+
+        # Course evaluation ends in 1 day.
+        cls.no_remind_course = mommy.make(
+                Course,
+                state='inEvaluation',
+                vote_start_date=cls.today - datetime.timedelta(days=2),
+                vote_end_date=cls.today + datetime.timedelta(days=1),
+                participants=[cls.user_not_to_remind])
+
+    def test_handle(self):
+        with patch('evap.evaluation.models.EmailTemplate.send_reminder_to_user') as mock:
+            management.call_command('send_reminders', 'daily')
+
+        self.assertEquals(mock.call_count, 2)
+        mock.assert_any_call(self.user_to_remind, first_due_in_days=2, due_courses=[(self.course1, 2)])
+        mock.assert_any_call(self.user_to_remind_twice, first_due_in_days=0, due_courses=[(self.course2, 0), (self.course1, 2)])
```

Comment:
...are there reasons to not split this into individual test methods? for each "type" of course one, and maybe one to ensure only one mail is sent in case multiple courses are affected or something like tha.